docs/guides/integration-options-private-operator.md

```diff
@@ -0,0 +1,102 @@
+
+## Private Operator Workflow
+
+The basic workflow for a Private Operator is as follows:
+
+1. On startup, the Private Operator goes through an attestation process with the <a href="../ref-info/glossary-uid#gl-core-service">Core</a> service. The attestation process verifies that the Operator is running in a secure trusted execution environment (TEE), and that the environment hasn't been tampered with.
+
+1. When the Operator passes the attestation process, the Core service gives the Private Operator secure S3 URLs for retrieving the information it needs for startup.
+
+1. The Private Operator retrieves the security information from Amazon S3 that it needs to process EUIDs, such as salts, encryption keys, and user opt-out records. For security details, see [Private Operator Security](#private-operator-security).
+
+1. If an Operator is restarted, it goes through the attestation process again, and retrieves a fresh set of security information.
+
+1. The Operator re-attests periodically with the Core service to ensure that it is still running in a protected environment. If attestation fails, the Operator shuts down.
+
+## Private Operator Security
+
+Each supported Private Operator implementation must meet rigorous security standards. Some security points include:
+
+- The Private Operator runs in a hardware-based trusted execution environment (TEE) hosted by a supported cloud provider listed in [Hosting Options for Private Operators](#hosting-options-for-private-operators).
+- The Private Operator must complete an attestation process before accessing the information needed to process EUIDs.
+- The information on S3 is encrypted at rest and also encrypted in transit through TLS. In addition, access is limited to only correctly authorized and attested Private Operators.
+- The information retrieved at startup is not stored locally at any point. It is only ever held in memory, and the Private Operator is running in a protected environment that makes it difficult for anyone running the Operator (such as an Administrator), as well as any external players, to see the data that's in memory.
+- The Private Operator never stores <Link href="../ref-info/glossary-uid#gl-personal-data">personal data</Link> that is sent for processing (email addresses). The data is only used within the enclave, to generate EUIDs, and is discarded immediately after processing.
+
+## Private Operator Limitations
+
+There are a couple of limitations to Private Operator functionality:
+- Private Operators do not currently support <Link href="../ref-info/glossary-uid#gl-client-side">client-side integration</Link>.
+- Private Operator updates are released three times per year; Public Operator updates are released on a more frequent cadence.
+
+## Getting Started
+
+To get started as a Private Operator, follow these steps:
+
+1. Request access to EUID by filling out the form on the [Request Access](/request-access) page.
+2. Decide which implementation option you want to use.
```

Comment:
We should probably change this as there is only one hosting option in EUID - AWS Nitro. Maybe just remove this line, and only have the "For details about ...." line